### PR TITLE
Fix RA infantry death anims playing too fast

### PR DESCRIPTION
**Pull request by reaperrr**
I noticed this literally years ago, but whenever it stood out to me I was to busy with testing something and forgot about it before I could file a PR.

Matches TD death anim speeds, is more consistent with most other animations and simply looks better this way.
Should be picked to prep, in my opinion.

### CODE REVIEW

File: mods/ra/sequences/infantry.yaml [hunk 1]
```diff
@@ -354,22 +372,28 @@ medi:
 		Tick: 120
 	die1:
 		Start: 193
-		Length: 7
+		Length: 8
```
Comment (reaperrr) [inline mods/ra/sequences/infantry.yaml:381]: Pretty sure 7 was a typo/error, so I sneaked this in as well.

File: mods/ra/sequences/infantry.yaml [hunk 2]
```diff
@@ -415,22 +439,28 @@ mech:
...
```
Comment (reaperrr) [inline mods/ra/sequences/infantry.yaml:448]: Same here.

**Review by dragunoff — CHANGES_REQUESTED** [review]
This indeed looks better in-game. It should be applied to the sniper as well. Perhaps you missed it.

**Review by Smittytron — DISMISSED** [review]
Looks good 👍

**Comment by reaperrr** [comment]
> It should be applied to the sniper as well. Perhaps you missed it.

I had thought we didn't use the Sniper anymore, but forgot to make sure. Fixed.

**Review by pchote — APPROVED** [review]
> I noticed this literally years ago, but whenever it stood out to me I was to busy with testing something and forgot about it before I could file a PR.

Likewise. Thanks for finally fixing this!

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/0d1d399a7a18acbd59e9681ed7bba124ddb7d1ec

de6a9dc8da

**Pull request merged**